gulp/tasks/webpack.js

```diff
@@ -4,8 +4,18 @@ const pi = require('gulp-load-plugins')();
 const path = require('path');
 const webpackStream = require('webpack-stream');
 const webpack2 = require('webpack');
+const HardSourceWebpackPlugin = require('hard-source-webpack-plugin');
+
 
 function webpack() {
+    const isDevelopment = config.env === 'development'
+    const plugins = isDevelopment ?
+        [new HardSourceWebpackPlugin()] :
```

Comment:
@RoyEJohnson Why was the `HardSourceWebpackPlugin` only included when in development mode?